### PR TITLE
147/show user name in the header after login using Okta in React & Angular app.

### PR DESCRIPTION
**Pull request by manjeetbeniwalinnostax**
**Description:**

1. Added username in Angular and React App for Okta.
2. Update styling of a Logout button added dropdown in Angular app.
3. Implemented username with the Logout button in React app.

### CODE REVIEW

File: src/templates/angular/src/app/app.module.ts [hunk 1]
```diff
@@ -7,17 +7,18 @@ import { SharedModule } from './shared/shared.module';
...
 <% if (isOkta) { %>
 const oktaAuth = new OktaAuth({
-issuer: '{Issuer URI}',
-clientId: '{Client ID }',
-redirectUri: window.location.origin+'/login/callback'});
+  issuer: environment.ANGULAR_APP_OKTA_ISSUER,
+  clientId: environment.ANGULAR_APP_OKTA_CLIENT_ID,
+  redirectUri: environment.ANGULAR_APP_API_URL});
 <% } %>
 @NgModule({
```
Comment (abhishek-innostax) [inline src/templates/angular/src/app/app.module.ts:23]: can start with new line ..

File: src/themeProviderTemplates/angular-themes/dark-theme.css [hunk 2]
```diff
@@ -43,6 +43,11 @@
 [data-theme='dark'] .navbar-toggler-icon {
   filter: brightness(0) invert(1);
 }
+/* Signout button styling */
```
Comment (abhishek-innostax) [inline src/themeProviderTemplates/angular-themes/dark-theme.css:46]: on line space as done on line  no - 25

File: src/templates/angular/src/environments/environment.ts [hunk 3]
```diff
@@ -3,7 +3,12 @@
 // The list of file replacements can be found in `angular.json`.
 
 export const environment = {
-  production: false
+  production: false,<% if(isOkta){ %>
```
Comment (abhishek-innostax) [inline src/templates/angular/src/environments/environment.ts:6]: if block can be put into next line ..

**Comment by manjeetbeniwalinnostax** [comment]
will resolve comments in the follow up PR

**Pull request merged**